dev/system-test/firestore.ts

```diff
@@ -1670,12 +1768,53 @@ describe('Query class', () => {
     expectDocs(res, {doc: 2}, {doc: 3}, {doc: 4});
   });
 
-  it('has offset() method', async () => {
+  it('has offset() method on get()', async () => {
     await addDocs({foo: 'a'}, {foo: 'b'});
     const res = await randomCol.orderBy('foo').offset(1).get();
     expectDocs(res, {foo: 'b'});
   });
 
+  it('has offset() method on stream()', done => {
```

Comment:
Offset method is on `Query`. Maybe consider rewording this test to `'can use offset() method with get()'`